spec/genre_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../classes/genre'
+
+describe Genre do
+  before :each do
+    @genre = Genre.new('country')
```

Comment:
The constructor of Genre changed. I'll approve it and update it later. Just to let you know.

Reply:
Thanks.